### PR TITLE
adding a python language peculiarity to the mix of language weirdness.

### PR DESCRIPTION
**Pull request by mcfarke311**
This should button up my thoughts on issue:#9 unless I find some more to add. Further weirdness with python could probably just be an intro to another week/lesson.

### CODE REVIEW

File: Lecture Material/Week 02 - HTML, css, javascript; Python Variables, Expressions, Statements/02 - HTML, CSS, and Javascript.ipynb [hunk 1]
```diff
@@ -402,7 +424,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.6"
+   "version": "3.6.8"
```
Comment (mertzjames) [inline Lecture Material/Week 02 - HTML, css, javascript; Python Variables, Expressions, Statements/02 - HTML, CSS, and Javascript.ipynb:427]: Due to the way that azure notebooks work (the system I'm using to deliver the lectures), this actually needs to be 3.6.6 for now.  Might be good to create an environment with that specific version of Python.

File: Lecture Material/Week 02 - HTML, css, javascript; Python Variables, Expressions, Statements/02 - HTML, CSS, and Javascript.ipynb [hunk 2]
```diff
@@ -296,9 +296,31 @@
    "source": [
     "### No Seriously, It's Really Weird:\n",
     "\n",
+    "Every language has it's peculiarities...\n",
```
Comment (mertzjames) [inline Lecture Material/Week 02 - HTML, css, javascript; Python Variables, Expressions, Statements/02 - HTML, CSS, and Javascript.ipynb:299]: I would actually prefer to add this to the lesson on Numbers and Math (or maybe a later one when we talk about comparisons).  I'm not sure it fits here in that the data will be retained by the reader for when it really counts.

**Review by mertzjames — CHANGES_REQUESTED** [review]
Great pull request!  and thank you for helping out!  There's a few things that should be addressed before I can approve.  Let me know if you have any questions about my comments.

File: Lecture Material/Week 02 - HTML, css, javascript; Python Variables, Expressions, Statements/02 - HTML, CSS, and Javascript.ipynb (hunk 2 above)
Comment (mcfarke311) [reply Lecture Material/Week 02 - HTML, css, javascript; Python Variables, Expressions, Statements/02 - HTML, CSS, and Javascript.ipynb:299]: makes sense. That is what I was wondering. I thought that was the spot that it fit best in the current lessons.

**Pull request closed without merging**